tool/tsh/access_request.go

```diff
@@ -378,37 +378,35 @@ func onRequestSearch(cf *CLIConf) error {
 	}
 
 	rows := [][]string{}
-	resourceIDs := []types.ResourceID{}
+	var resourceIDs []string
 	for _, resource := range resources {
-		resourceIDs = append(resourceIDs, types.ResourceID{
+		resourceID := types.ResourceIDToString(types.ResourceID{
 			ClusterName: clusterName,
 			Kind:        resource.GetKind(),
 			Name:        resource.GetName(),
 		})
+		resourceIDs = append(resourceIDs, resourceID)
 		hostName := ""
 		if r, ok := resource.(interface{ GetHostname() string }); ok {
 			hostName = r.GetHostname()
 		}
 		rows = append(rows, []string{
-			resource.GetKind(),
+			resource.GetName(),
 			hostName,
 			sortedLabels(resource.GetAllLabels()),
-			resource.GetName(),
+			resourceID,
 		})
 	}
-	table := asciitable.MakeTableWithTruncatedColumn([]string{"Kind", "Hostname", "Labels", "ID"}, rows, "Labels")
+	table := asciitable.MakeTableWithTruncatedColumn([]string{"Name", "Hostname", "Labels", "ID"}, rows, "Labels")
```

Comment:
nit: The last column here is `ID`, but it is used for parameter `--resource`. It would be better if we can make the naming more consistent or more obvious they are related.

Reply:
Good point, updated the header to `Resource ID`